crates/compilers/src/compilers/solc/mod.rs

```diff
@@ -188,6 +190,9 @@ pub struct SolcSettings {
     /// Additional CLI args configuration
     #[serde(flatten)]
     pub cli_settings: CliSettings,
+    /// Additional custom resolc compiler settings
+    #[serde(flatten)]
+    pub extra_settings: ResolcSettings,
```

Comment:
dq: but why `extra_settings` when it's `resolc` settings?

Reply:
We can rename it, I do not like this that we have `ResolcSettings` in `SolcSettings`, but as you saw in [take #1 PR](https://github.com/paritytech/foundry-compilers-polkadot/pull/24) there popups another issues and probably for now this is a better approach until we have our `ResolcMultiCompiler`